src/main/java/exam2/lotto/LottoNumber.java

```diff
@@ -0,0 +1,94 @@
+package exam2.lotto;
+
+import java.util.Objects;
+
+public class LottoNumber implements Comparable<LottoNumber> {
+
+    /**
+     * 로또 시작 번호
+     */
+    public static final int LOTTO_START_NUMBER = 1;
+
+    /**
+     * 로또 마지막 번호
+     */
+    public static final int LOTTO_END_NUMBER = 45;
+
+    private int number;
+
+
+    /**
+     * 로또 번호 유효성 체크
+     *
+     * @param number
+     */
+    public LottoNumber(String number) {
+        this.number = checkValidRange(number);
+    }
+
+    public LottoNumber(int number) {
+        this.number = checkValidRange(number);
+    }
+
+    /**
+     * 로또 범위가 맞는지 확인합니다.
+     *
+     * @param inputNumber
+     * @return
+     */
+    public int checkValidRange(String inputNumber) {
+        int lottoNumber = 0;
+
+        try {
+            lottoNumber = Integer.parseInt(inputNumber.trim());
+        } catch (NumberFormatException e) {
+            throw new NumberFormatException("숫자가 아닌 값을 입력할 수 없습니다");
+        }
+
+        if (!(lottoNumber <= LOTTO_END_NUMBER && lottoNumber >= LOTTO_START_NUMBER)) {
+            throw new IllegalArgumentException("1 ~ 45 사이 값을 입력 해주세요");
+        }
+
+        return Integer.parseInt(inputNumber.trim());
```

Comment:
```suggestion
        try  {
             return Integer.parseInt(inputNumber.trim());
        } catch (NumberFormatException e) {
            throw new NumberFormatException("숫자가 아닌 값을 입력할 수 없습니다");
        }
```
생성자를 한쪽에서 관리하면 이렇게 제거 할 수도 있을꺼 같아요. 
그러면 해당 메서드의 이름은 문자인지 아닌지 체크를 하게 될꺼 같네요.

Reply:
수정했습니다! 저렇게 수정하니까 훨씬 코드가 간결해져서 좋네요 ^ㅡ^